### PR TITLE
Update reference items in API menu for v3.0.0

### PR DESCRIPTION
**Pull request by cybette**
Changes in API doc structure in new version needs to be reflected on the MIQ website.

http://talk.manageiq.org/t/api-refrerence-404-on-some-urls/3449

### CODE REVIEW

**Comment by abellotti** [comment]
There's new files that are not reflect here. that'd be in the _topic_map.yml file change as per https://github.com/ManageIQ/manageiq_docs/pull/676/files#diff-9ff4392b3b4341f06984a7cd81583cd5

**Comment by cybette** [comment]
Thanks @abellotti for pointing me to the right diff. Please check now.

**Comment by cybette** [comment]
Thank you @abellotti for reviewing!

@epwinchell could you please give it a once-over and merge? Thanks!

File: site/_data/menus/api_menu.yml [hunk 1]
```diff
@@ -18,39 +18,67 @@
...
   - title: Automate Model
     path: "/docs/reference/latest/api/reference/automate"
   - title: Automate Domains
     path: "/docs/reference/latest/api/reference/automate_domains"
   - title: Automation Requests
     path: "/docs/reference/latest/api/reference/automation_requests"
-  - title: Blueprint Support
-    path: "/docs/reference/latest/api/reference/blueprints"
+  - title: Automate Workspaces
+    path: "/docs/reference/latest/api/reference/automate_workspaces"
```
Comment (epwinchell) [inline site/_data/menus/api_menu.yml]: @cybette This link seems broken

File: site/_data/menus/api_menu.yml [hunk 2]
```diff
@@ -18,39 +18,67 @@
...
   - title: Categories & Tags
     path: "/docs/reference/latest/api/reference/categories_tags"
   - title: Chargebacks & Rates
     path: "/docs/reference/latest/api/reference/chargebacks_rates"
+  - title: Cloud Volumes
```
Comment (epwinchell) [inline site/_data/menus/api_menu.yml]: @cybette This one is broken too.

File: site/_data/menus/api_menu.yml [hunk 3]
```diff
@@ -18,39 +18,67 @@
...
+    path: "/docs/reference/latest/api/reference/cloud_volumes"
+  - title: Conditions
+    path: "/docs/reference/latest/api/reference/conditions"
+  - title: Configuration Script Source Management
+    path: "/docs/reference/latest/api/reference/configuration_script_sources"
+  - title: Custom Actions
+    path: "/docs/reference/latest/api/reference/custom_actions"
+  - title: Custom Attributes
+    path: "/docs/reference/latest/api/reference/custom_attributes"
+  - title: Custom Button Management
```
Comment (epwinchell) [inline site/_data/menus/api_menu.yml]: @cybette and this one

Comment (epwinchell) [reply site/_data/menus/api_menu.yml]: Many of the titles from here down are broken.

**Comment by epwinchell** [comment]
@skateman Can you see what's going on here? Half the additions are broken links. Thanks.

**Comment by cybette** [comment]
@epwinchell if you're testing locally, did you rebuild the API docs? all these links are live at least, e.g. [Cloud Volumes](http://manageiq.org/docs/reference/latest/api/reference/cloud_volumes), [Automate Workspaces](http://manageiq.org/docs/reference/latest/api/reference/automate_workspaces) etc.

**Comment by epwinchell** [comment]
@cybette This works fine for @skateman, locally - merging.

**Pull request merged**